src/apps/HeyGPT/HeyGPT/Models/SimpleMessagePrompt.cs

```diff
@@ -0,0 +1,24 @@
+﻿#nullable disable
+
+using System.Text.Json;
+
+namespace HeyGPT.App.Models;
+
+public readonly record struct SimpleMessagePrompt
```

Comment:
Very simple data structure for the UX to read prompts from file or protocols.   Reminds me I should use this for the protocol handler as well.